index.html

```diff
@@ -3,10 +3,155 @@
+        <li class="mobile-link" onclick="closeNave();"><a href="#aboutme-container" >Program</a></li>
+        <li class="mobile-link" onclick="closeNave();"><a href="#contact">News</a></li>
+    </section>
+
+    <section class="home-page" id="home-page" >
+      <div class="containerHome-Page">
+        <div>
+          <p class="greetingOne">"Hello, Sharing World!"</p>
+          <h1 class="home-title" id="home-title" >Urban Art Festival Creativity in the Streets 2023</h1>
+        </div>
+        <div class="home-Description">
+          <p>
+            "Join us at the exhilarating 'Creativity in the Streets' Urban Art Festival, where the most talented street artists come together to transform the streets of our city into an outdoor art gallery. This annual festival celebrates urban culture, artistic expression, and the creative community in a vibrant and unique atmosphere."
+          </p>
+        </div>
+        <div>
+          <p class="home-date">2023.07.03(THU) ~ 12(WEN)</p>
+          <p class="home-location-info">@StudentsToWorkd Collage-EEUU</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="activities" id="activities">
+    <div class="overlay">
+      <div class="activities-subtitle">
+        <h2> Main Program </h2>
+        <hr>
+      </div>
+      <div class="containerBoxs">
+        <div class="boxs">
+          <img src="/images/nuevo-mensaje.png" class="imagesActivity" alt="" srcset="">
+          <h4>Forum</h4>
+          <p>Enjoy taking some time Take the time to share your thoughts and opinions with experts for each topic.</p>
+        </div>
+        <div class="boxs">
+          <img src="/images/taller.png" class="imagesActivity" alt="" srcset="">
+          <h4>Advisory</h4>
+          <p>Get free advice and recommendations from the most experts in the field, negotiate according to your interests.</p>
+        </div>
+        <div class="boxs">
+          <img src="/images/comida.png" class="imagesActivity" alt="" srcset="">
+          <h4>Aperitif</h4>
+          <p>Enjoy a delicious appetizer while you enjoy observing the greatest works of great artists of the genre.</p>
+        </div>
+        <div class="boxs">
+          <img src="/images/tendencia.png" class="imagesActivity" alt="" srcset="">
+          <h4>Trends</h4>
+          <p>Find out about the latest trends, update your knowledge, get ideas, find best and most detailed information.</p>
+        </div>
+        <div class="boxs">
+          <img src="/images/exhibicion.png" class="imagesActivity" alt="" srcset="">
+          <h4>Exhibition</h4>
+          <p>You can enjoy a wide gallery of exhibitions, where you can find a very varied range of works made in different cultures.</p>
+        </div>  
+      </div>
+      <div class="containerButton">
+        <button type="button" class="Button" id="Button">Join Urban Art Festival</button>
```

Comment:
- [ ] please ensure the text under the cards is underline like the previous reviewer noted. yours has a line under.
yours
![Screenshot (1305)](https://github.com/Fernando426/CapstoneN01/assets/65298048/21223f4b-849f-4c84-8305-2a4edbd8dc2f)
original
![Screenshot (1306)](https://github.com/Fernando426/CapstoneN01/assets/65298048/9c37e4d4-2fc6-47f3-9e60-7741cd8cbc6b)